lib/tito.rb

```diff
@@ -0,0 +1,29 @@
+require "httparty"
+require_relative "tito/event"
+require_relative "tito/null_event"
+
+class Tito
+  include HTTParty
+
+  base_uri "https://api.tito.io/railsbridge-boston"
+
+  def event_list
+    if response.success? && !response.empty?
+      response["events"].map do |event|
+        Event.new(event)
+      end
+    else
+      [NullEvent.new]
+    end
+  end
+
+  private
+
+  def options
+    { query: { auth_token: ENV.fetch("TITO_AUTH_TOKEN") } }
+  end
+
+  def response
+    self.class.get("/events.json", options)
```

Comment:
I think I would move this method above options. My first thing was to look below this method for the `options` method.